pkg/authorization/apis/authorization/v1/conversion.go

```diff
@@ -394,15 +394,32 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 		return err
 	}
 
-	if err := scheme.AddFieldLabelConversionFunc("v1", "PolicyBinding",
-		apihelpers.GetFieldLabelConversionFunc(newer.PolicyBindingToSelectableFields(&newer.PolicyBinding{}), nil),
-	); err != nil {
+	return nil
+}
+
+func addLegacyFieldSelectorKeyConversions(scheme *runtime.Scheme) error {
+	if err := scheme.AddFieldLabelConversionFunc(LegacySchemeGroupVersion.String(), "PolicyBinding", legacyPolicyBindingFieldSelectorKeyConversionFunc); err != nil {
 		return err
 	}
+	return nil
+}
 
+func addFieldSelectorKeyConversions(scheme *runtime.Scheme) error {
 	return nil
```

Comment:
Why this is empty?

Reply:
> Why this is empty?

We don't have any custom field selectors.  Yet.  I'm placing the skeleton here so people can more easily figure out what to do.  This isn't really obvious.